tools/wpt/wpt.py

```diff
@@ -95,7 +95,8 @@ def create_complete_parser():
     for command in commands:
         props = commands[command]
 
-        if props["virtualenv"]:
+        if props["virtualenv"] and (
+            (not props["py3only"]) or sys.version_info.major == 3):
```

Comment:
Otherwise, `pip install aioquic` will cause a fatal error in Python 2.

Reply:
Also, @jgraham I think this is the indentation style you prefer, but `flake8` is sad:
https://community-tc.services.mozilla.com/tasks/I9HcCed5SSqARezBCYTroQ/runs/0/logs/https%3A%2F%2Fcommunity-tc.services.mozilla.com%2Fapi%2Fqueue%2Fv1%2Ftask%2FI9HcCed5SSqARezBCYTroQ%2Fruns%2F0%2Fartifacts%2Fpublic%2Flogs%2Flive.log#L356

Reply:
My preferred style would be:

```py
        if (props["virtualenv"] and
            (not props["py3only"] or sys.version_info.major == 3)):
```